README.md

```diff
@@ -18,11 +18,16 @@ The master branch is in version 1.5.0-SNAPSHOT:
 
 ### Dependency Information
 
+Alluxio provides several different client interfaces. The Alluxio file system interface provided by
+`alluxio-core-client-fs` is suggested for performance and flexibility. If you want to use other
+interfaces, include the appropriate client artifact. For example, `alluxio-core-client-hdfs` provides
+a client implementing HDFS's file system API.
+
 #### Apache Maven
 ```xml
 <dependency>
   <groupId>org.alluxio</groupId>
-  <artifactId>alluxio-core-client</artifactId>
+  <artifactId>alluxio-core-client-fs</artifactId>
```

Comment:
emm, 1.4.0 does not have this artifact, so this doc will be not accurate before 1.5.0 released, I guess